src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -1360,64 +1360,53 @@ void Instance::OnFailSafeTimerExpired()
     }
 }
 
-CHIP_ERROR Instance::EnumerateAcceptedCommands(const ConcreteClusterPath & cluster, CommandIdCallback callback, void * context)
+CHIP_ERROR Instance::EnumerateAcceptedCommands(const ConcreteClusterPath & cluster,
+                                               DataModel::ListBuilder<DataModel::AcceptedCommandEntry> & builder)
 {
     using namespace Clusters::NetworkCommissioning::Commands;
+    using QF                            = DataModel::CommandQualityFlags;
+    static const auto kDefaultFlags     = chip::BitFlags<QF>(QF::kTimed, QF::kLargeMessage, QF::kFabricScoped);
+    static const auto kDefaultPrivilege = chip::Access::Privilege::kOperate;
 
-    if (mFeatureFlags.Has(Feature::kThreadNetworkInterface))
+    bool hasNet  = mFeatureFlags.Has(Feature::kThreadNetworkInterface);
+    bool hasWifi = mFeatureFlags.Has(Feature::kWiFiNetworkInterface);
+    bool hasCred = mFeatureFlags.Has(Feature::kPerDeviceCredentials);
+    auto netId   = hasNet ? AddOrUpdateThreadNetwork::Id : AddOrUpdateWiFiNetwork::Id;
+
+    static const DataModel::AcceptedCommandEntry commands[6] = {
```

Comment:
this feels a bit hard to read given 2 elements per line. could you re-format (e.g. adding a trailing `//` will force the formatter not to join lines.